src/alez05/section-tag.pug

```diff
@@ -0,0 +1,20 @@
+html
+  head
+    title Whole Cake Island
+  body
+    h1 The section element
+    p This is it
+    section
+    h2 WWF History
+    p The World Wide Fund for Nature (WWF) is an international organization working on issues regarding the conservation, research and restoration of the environment, formerly named the World Wildlife Fund. WWF was founded in 1961.
```

Comment:
8 and 9 should be children to **section**, that's why we use the tag, to wrap other tags